kythe/typescript/indexer.ts

```diff
@@ -965,6 +993,60 @@ class Visitor {
     return undefined;
   }
 
+  getSymbolAndVNameForFunctionDeclaration(node: ts.FunctionLikeDeclaration):
+      [ts.Symbol|undefined, VName|undefined] {
+    let context: Context|undefined = undefined;
+    if (ts.isGetAccessor(node)) {
+      context = Context.Getter;
+    } else if (ts.isSetAccessor(node)) {
+      context = Context.Setter;
+    }
+    if (node.name) {
+      const sym = this.host.getSymbolAtLocation(node.name);
+      if (!sym) { return [undefined, undefined]; }
+      const vname =
+          this.host.getSymbolName(sym, TSNamespace.VALUE, context);
+      return [sym, vname];
+    } else {
+      // TODO: choose VName for anonymous functions and return symbol
+      return [undefined, this.newVName('TODO', 'TODOPath')];
```

Comment:
My above suggestion would make this just

```
return {vname: ...}
```

Reply:
Done.